pulp_2to3_migration/app/plugin/docker/utils.py

```diff
@@ -0,0 +1,21 @@
+from . import pulp2_models
+
+from pulp_2to3_migration.pulp2 import connection
+
+
+connection.initialize()
```

Comment:
It's surprising that it's needed.
I guess it's because you call it at the time the migrator is evaluated. If we store only the reference to a function I think this additional mongo connection won't be needed.

Reply:
correct